CHANGELOG/v1.7.0-beta.0.md

```diff
@@ -0,0 +1,397 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+
+* Enable kubeadm feature gates mutation
+
+## Deprecation Warning
+
+- The API version `v1alpha4` has been completely removed in this release.
```

Comment:
Not blocking for the beta release, but I think we should review this statement before GA
cc @sbueringer

Reply:
Yup this is not true anymore